plugins/ruby-foreman-tasks/debian/control

```diff
@@ -2,7 +2,7 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.7)
+Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.9)
```

Comment:
I will have new foreman-tasks version, that will need this version as well, so I think we can leave it here.